.github/workflows/build_and_push_to_docker.yml

```diff
@@ -68,7 +68,7 @@ jobs:
           export LABEL_PREFIX=${{ matrix.BASE_IMAGE_TAG }}
         fi
 
-        docker build docker/py --tag $DOCKER_NAME-py:$LABEL_PREFIX-$LABEL --build-arg BASE_IMAGE="jupyter/base-notebook:${{ matrix.BASE_IMAGE_TAG }}"
+        docker build . -f docker/py/Dockerfile --tag $DOCKER_NAME-py:$LABEL_PREFIX-$LABEL --build-arg BASE_IMAGE="jupyter/base-notebook:${{ matrix.BASE_IMAGE_TAG }}"
```

Comment:
See my other comment. I think this should remain as is. The build context should be the location where the Dockerfile for each image is. With this change we make the context be the root of the repository and then we have to include/change paths to every file referenced in the Dockerfile relative to the root of the repository.